poetry/utils/helpers.py

```diff
@@ -27,19 +27,18 @@ def normalize_version(version):  # type: (str) -> str
     return str(Version(version))
 
 
+def del_ro(action, name, exc):
```

Comment:
How about prepending the function name with a `_`/`__` or namespacing it inside `temporary_directory` function, so it doesn't get 'exported' from the helpers file?

Reply:
Good idea.  Honestly, I'm surprised any of these helpers are "public".